notebooks/official/workbench/fraud_detection/fraud-detection-model.ipynb

```diff
@@ -219,11 +219,12 @@
       "source": [
```

Comment:
<p>remove table of contents</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/821/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>